PICS.xcodeproj/project.pbxproj

```diff
@@ -1227,7 +1239,7 @@
 				BUNDLE_LOADER = "$(TEST_HOST)";
 				CODE_SIGN_STYLE = Automatic;
 				CURRENT_PROJECT_VERSION = 1;
-				DEVELOPMENT_TEAM = 637867499T;
+				DEVELOPMENT_TEAM = "";
```

Comment:
For the release, we might want to keep the development_team? Otherwise there might be licensing issues?